### PR TITLE
HR-8525:add TokenParameters unique

### PR DESCRIPTION
**Pull request by samsunxuguang**

### CODE REVIEW

File: src/Gateways/PorticoConnector.ts [hunk 1]
```diff
@@ -417,6 +417,11 @@ export class PorticoConnector extends XmlGateway implements IPaymentGateway {
       subElement(cardData, "TokenRequest").append(
         cData(builder.requestMultiUseToken ? "Y" : "N"),
       );
+
+      if(builder.tokenUqinue){
+        subElement(subElement(cardData, "TokenParameters"),
```
Comment (fisher-he) [inline src/Gateways/PorticoConnector.ts:422]: Suggest it's need to extract the separate elements.

File: src/Builders/AuthorizationBuilder.ts [hunk 2]
```diff
@@ -644,6 +645,13 @@ export class AuthorizationBuilder extends TransactionBuilder<Transaction> {
     return this;
   }
 
+  public widthTokenUqinue(tokenUqinue?: boolean) {
```
[the remark this answers is not in the document]
Comment (samsunxuguang) [reply src/Builders/AuthorizationBuilder.ts]: Resolved

File: src/Builders/AuthorizationBuilder.ts [hunk 3]
```diff
@@ -55,6 +55,7 @@ export class AuthorizationBuilder extends TransactionBuilder<Transaction> {
   public scheduleId: string;
   public shippingAddress: Address;
   public timestamp: string;
+  public tokenUqinue: boolean;
```
[the remark this answers is not in the document]
Comment (samsunxuguang) [reply src/Builders/AuthorizationBuilder.ts]: Resolved

File: src/Gateways/PorticoConnector.ts (hunk 1 above)
Comment (samsunxuguang) [reply src/Gateways/PorticoConnector.ts]: Resolved

**Review by fisher-he — APPROVED** [review]

**Review by liuchuck — APPROVED** [review]

**Pull request merged**